eng/scripts/Language-Settings.ps1

```diff
@@ -348,7 +369,45 @@ function Get-DocsCiLine ($item) {
   return $line
 }
 
+function EnsureCustomSource($package) {
+  # $PackageSourceOverride is a global variable provided in 
+  # Update-DocsMsPackages.ps1. Its value can set a "customSource" property.
+  # If it is empty then the property is not overridden.
+  if (!$PackageSourceOverride) {
```

Comment:
You might want to use this `Get-Variable -Name 'PackageSourceOverride' -ValueOnly -ErrorAction 'Ignore'`